app/frontend/src/features/profile/constants.ts

```diff
@@ -16,6 +16,20 @@ export const NO_MEETUP = "Cannot meet up";
 
 const UNSURE = "Ask me";
 
+// User reporting
+export const MORE_PROFILE_ACTIONS = "...";
+export const MORE_PROFILE_ACTIONS_A11Y_TEXT = "More profile actions";
+export const REPORT_USER = "Report this user";
+export const REPORT_REASON = "Reason";
+export const REPORT_DETAILS = "Details";
+export const SEND = "Send";
+export const CANCEL = "Cancel";
+export const getReportUserExplainer = (name: string) =>
```

Comment:
why `get...`?

Reply:
It's a standard naming convention I got used to when something is a function rather than a simple property access. And I guess these are basically dynamic property getters so probably doesn't hurt to encode that meaning.